### PR TITLE
Add publish to pypi section to github action

### PR DESCRIPTION
**Pull request by jonathancychow**

### CODE REVIEW

File: .github/workflows/build-and-publish.yml [hunk 1]
```diff
@@ -41,9 +41,16 @@ jobs:
...
           repo_token: "${{ secrets.GITHUB_TOKEN }}"
           files: |
             dist/*.tar.gz
             dist/*.whl
+
+      - name: Publish distribution to PyPI
+        uses: pypa/gh-action-pypi-publish@master
+        with:
+          user: __token__
+          password: ${{ secrets.PYPI_API_TOKEN }}
```
Comment (jonathancychow) [inline .github/workflows/build-and-publish.yml:56]: I got a token generated for this project, I can pass it to you @Stedders then see if github action works with pypi or not?

**Review by dannystaple — COMMENTED** [review]
Looks good - once the token is in place.

**Review by dannystaple — APPROVED** [review]

File: .github/workflows/build-and-publish.yml (hunk 1 above)
Comment (Stedders) [reply .github/workflows/build-and-publish.yml:56]: Sounds good. Shall we connect on Saturday?

I have questions about the token. Is it associated with your account? If so I would prefer a generic one associated with a coderdojo account.

Comment (jonathancychow) [reply .github/workflows/build-and-publish.yml:56]: 👍 
It is associated with [this particular project](https://pypi.org/project/coder-dojo-common-python/) in my account, I just wanted to get pypi up and running first so people can actually pip install thing. 

I think the right thing to do is probably link everything back to the [coderdojo account](https://github.com/coder-dojo-ham). I'll have a look.

Comment (Stedders) [reply .github/workflows/build-and-publish.yml:56]: Ok. That's fine. Feel free to merge. I won't be able to look at this again until Wednesday.

Comment (Stedders) [reply .github/workflows/build-and-publish.yml:56]: Oh. You need to give me the token.

LOL

**Pull request merged**